nginx/nginx.conf

```diff
@@ -44,7 +44,7 @@ http {
 	# it could be "close" to close a keepalive connection
 	proxy_set_header Connection '';
 	proxy_set_header Host $host;
-	proxy_set_header X-Real-IP $remote_addr;
+	proxy_set_header X-Real-IP "";
```

Comment:
```suggestion
```

Shouldn't we just remove this line?

Reply:
Removing this line leaves header untouched if it already exists. 
In this case there is probability that header will be accidentally used by backend in future.